website/docs/addon_slack.md

```diff
@@ -15,83 +15,130 @@ import Tabs from '@theme/Tabs';
 
-- family: 'render'
-- host: 'Maya'
+- For all projects:
+  - `ayon+settings://slack/token`
+- For a specific project, replace {PROJECT_NAME} with your project's name:
+  - `ayon+settings://slack/token?project={PROJECT_NAME}`
 
-### Messages to channels
+You'll find this token in the Slack dashboard once the app is installed. Remember, you can create separate tokens for use with different projects.
 
-#### Channels
-Multiple messages could be delivered to one or multiple channels, by default app allows Slack bot
-to send messages to 'public' channels (eg. bot doesn't need to join the channel first).
+![Get token](assets/slack/slack_token.png)
 
-![Configure module](assets/slack_project.png)
+### Profiles
 
-#### Upload thumbnail
-Integration can upload 'thumbnail' file (if present in an instance), for that bot must be 
-manually added to each target channel by Slack admin first!
-(In target channel write: ```/invite @ayonNotifier``)
+![](assets/slack/profiles.png)
 
-#### Upload review
-Integration can upload 'review' file (if present in an instance), for that bot must be 
-manually added to each target channel by Slack admin first!
-(In target channel write: ```/invite @ayonNotifier``)
+Profiles determine when notifications are triggered. You can configure one or several profiles based on `Product types`, `Hosts`, `Task types`, and `Task names` or `Product names` (with regex support).
 
-This option is limited by `Upload review maximum file size` value in `Profiles` section. It might make sense to limit uploading
-of reviews only up to certain size. If the review file hits that limit, only link to studio accessible location will be inserted instead.
+For example, to receive notifications when a render is published from Maya, the settings would be:
 
-Burnin version of the review (usually .mp4) is preferred if present.
+- Product type: 'render'
+- Host: 'Maya'
 
+### Messages to channels
+
+![Configure module](assets/slack/messages_to_channels.png)
+
+- **Channels:**
+Specify one or more channels where you'd like the plugin to send messages.
+:::info
+Please note that the Slack bot can access public channels by default, so there's no need for it to join them first.
+:::
+- **Upload thumbnail** & **Upload review:**
+The plugin is capable of uploading a 'thumbnail' or 'review' from an instance (if they are present in an instance). However, your Slack bot must be manually invited to each target channel by a Slack admin.
+:::tip
+To add your bot to a target channel, simply mention it in the message field with the command: `/invite @ayonNotifier`.
+:::
+- **Message:**
+Message content can use template keys (see [Available template keys](admin_settings_project_anatomy#available-template-keys)).
+Few keys also have Capitalized and UPPERCASE format. Values will be modified accordingly. e.g. `{folder[name]}` ➜ "Gun", `{product}` ➜ "RENDER".
+:::tip
+We have one additional implemented key `{review_filepath}`
+:::
```

Comment:
```suggestion
Specify one or more channels where you'd like the plugin to send messages.

:::info
Please note that the Slack bot can access public channels by default, so there's no need for it to join them first.
:::

- **Upload thumbnail** & **Upload review:**
The plugin is capable of uploading a 'thumbnail' or 'review' from an instance (if they are present in an instance). However, your Slack bot must be manually invited to each target channel by a Slack admin.

:::tip
To add your bot to a target channel, simply mention it in the message field with the command: `/invite @ayonNotifier`.
:::

- **Message:**
Message content can use template keys (see [Available template keys](admin_settings_project_anatomy#available-template-keys)).
Few keys also have Capitalized and UPPERCASE format. Values will be modified accordingly. e.g. `{folder[name]}` ➜ "Gun", `{product}` ➜ "RENDER".

:::tip
We have one additional implemented key `{review_filepath}`
:::
```

Cosmetics...

It would be nice to maybe provide an example as to what `{review_filepath}` resolves to? Is it the published path from the machine emitting the signal? (this may be nice info for e.g. studios using site-sync setups) or is it a rootless path, etc?

Reply:
I've added an example here https://github.com/ynput/ayon-documentation/pull/218/commits/b91c4749cfb3e59fddc3a7a933310f161ebfda47 (reused an existing example),
I'm not sure how it works with sitesync or rootless paths. maybe @kalisp can help providing this example/info.

Reply:
Feel free to resolve if you feel this has been fixed. 👍